manifests/i/IDMComputerSolutions,Inc/UltraEdit/31.0.35/IDMComputerSolutions,Inc.UltraEdit.installer.yaml

```diff
@@ -10,11 +10,49 @@ Installers:
   InstallerUrl: https://downloads.ultraedit.com/main/ue/win/ue_english_64.msi
   InstallerSha256: 7E077C563FA4C0509096595A3E4B3E494534FBB6588CAF5BCBDDFBBEDA3E94E6
   ProductCode: '{32AE92AA-6BFE-1014-A875-D17E057FC493}'
+  AppsAndFeaturesEntries:
+    - DisplayName: UltraEdit
+      DisplayVersion: 31.0.35

```

Comment:
I'm curious if we really need this (since it matches the PackageVersion). Or is it that you want to be extra sure since we're creating the version mapping anyways by using DisplayVersion in another installer node

Reply:
I wanted to be extra sure that we are creating the mapping, especially due to 
* https://github.com/microsoft/winget-cli/issues/2503